src/boot/lib/tensor.ml

```diff
@@ -121,6 +136,16 @@ module Generic : GENERIC = struct
       t.data.(linear_idx) <- v
     else raise (Invalid_argument "Tensor.Op_mseq_generic.set_exn")
 
+  let linear_get_exn t linear_idx =
+    if linear_idx >= 0 && linear_idx < t.size then
+      t.data.(linear_idx + t.stride)
```

Comment:
Do you mind renaming `stride` to lets say `offset` in this PR as well? It would better describe what it really is.

Reply:
Done!